backend/telemetry/src/main/java/org/sonarsource/sonarlint/core/telemetry/payload/ShareConnectedModePayload.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * SonarLint Core - Telemetry
+ * Copyright (C) 2016-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.telemetry.payload;
+
+import com.google.gson.annotations.SerializedName;
+import javax.annotation.Nullable;
+
+public class ShareConnectedModePayload {
+
+  @SerializedName("manual_added_bindings_count")
+  public final Integer manualAddedBindingsCount;
+
+  @SerializedName("imported_added_bindings_count")
```

Comment:
minor note - I think `imported_bindings_count` & `auto_bindings_count` would be enough and also in line with what DPO [expects](https://docs.google.com/spreadsheets/d/1oIPd5dZrjWqtujEQ5hQIJ_fVe3heCUIzmNmEaoAC_W4/edit#gid=368366020)

Reply:
DPO needs the total binding count, which to get we need to add the manual binding counts. I don't think we can skip it, WDYT?

Reply:
Yes yes for sure, I just meant for the naming `imported_added_bindings_count` -> `imported_bindings_count`

Reply:
oh I see 😄